libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,205 @@
+    secureTransactionWrapper.append(secureElement, label);
+    return secureTransactionWrapper;
+  };
+
+  const createCheckbox = (checkBoxText) => {
+    cardFooter.querySelector('hr')?.remove();
+    const container = createTag('label', { class: 'checkbox-container' });
+    const input = createTag('input', { id: 'alt-cta', type: 'checkbox' });
+    const checkmark = createTag('span', { class: 'checkmark' });
+    const label = createTag('span', { class: 'checkbox-label' }, checkBoxText.innerHTML);
+    container.append(input, checkmark, label);
+    return container;
+  };
+
+  const decorateAlternativeCta = () => {
+    const toggleButtonActiveState = (buttonToAdd, buttonToRemove) => {
+      buttonToAdd.classList.add('button--inactive');
+      buttonToRemove.classList.remove('button--inactive');
+    };
+    const cardFooterRow = el.querySelector('.consonant-CardFooter-row');
+    const secureTransactionSign = el.querySelector('h6[id^="secure-transaction"]');
+    if (secureTransactionSign) {
+      const standardWrapper = createTag('div', { class: 'standard-wrapper' });
+      const secureTransactionWrapper = decorateWithSecureTransactionSign(secureTransactionSign);
+      standardWrapper.append(secureTransactionWrapper, cardFooterRow);
+      cardFooter?.append(standardWrapper);
+      secureTransactionSign.remove();
+    }
+
+    const altCta = Array.from(el.querySelectorAll('div > div[data-align="center"][data-valign="middle"]')).filter((data) => data.textContent?.trim() === 'alt-cta');
+    const altCtaMetaData = altCta[0]?.parentElement?.nextElementSibling?.querySelectorAll('div > div');
+    const altCtaRegex = /href="([^"]*)"/g;
+    if (altCtaMetaData?.length === 2 && altCtaMetaData[1]?.innerHTML?.match(altCtaRegex)) {
+      const originalCtaButton = cardFooterRow.querySelector('.consonant-CardFooter-cell--right');
+      const altCtaButton = originalCtaButton.cloneNode(true);
+      const checkboxContainer = createCheckbox(altCtaMetaData[0]);
+
+      altCtaButton.innerHTML = altCtaMetaData[1].innerHTML;
+      altCtaButton.classList.add('button--inactive');
+      checkboxContainer.querySelector('input[type="checkbox"]').addEventListener('change', (event) => {
+        if (event.target.checked) {
+          toggleButtonActiveState(originalCtaButton, altCtaButton);
+        } else {
+          toggleButtonActiveState(altCtaButton, originalCtaButton);
+        }
+      });
+      cardFooterRow.append(altCtaButton);
+      cardFooter.prepend(checkboxContainer);
+      altCta[0].parentNode.remove();
+      altCtaMetaData[0].parentNode.remove();
+    }
+  };
+
+  decorateAlternativeCta();
+  cardFooter.querySelectorAll('.consonant-CardFooter-cell').forEach((cell) => cell.classList.add(`consonant-${cardType}-cell`));
+};
+
+const addInner = (el, cardType, merchCard) => {
+  const titles = Array.from(el.querySelectorAll('h1, h2, h3, h4, h5, h6'));
```

Comment:
```suggestion
  const titles = [...el.querySelectorAll('h1, h2, h3, h4, h5, h6')];
```
nit. a 'shorter' way of the same thing